src/integrationTest/java/com/mongodb/kafka/connect/mongodb/ChangeStreamOperations.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.mongodb.kafka.connect.mongodb;
+
+import static java.util.stream.IntStream.rangeClosed;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.bson.Document;
+
+public class ChangeStreamOperations {
```

Comment:
Test data classes for the change stream events, drop, dropDatabase, insert (including _id) - so we can assert between what we expect to be on the topic and what was actually on the topic.